{electron-source.electron_37,electron_37-bin,electron-chromedriver_37}: init at 37.1.0

TomaSajt opened the pull request:
https://github.com/electron/electron/releases/tag/v37.0.0
https://github.com/electron/electron/releases/tag/v37.1.0

Closes https://github.com/NixOS/nixpkgs/issues/420228

I don't have the time to wait out the entire build, but it reached `1000/43620`. I doubt there will be any issues with the build, but who knows...

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md), [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md), [maintainers/README.md](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) and other contributing documentation in corresponding paths.

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

TomaSajt committed:
electron_37-bin: init at 37.1.0

- Changelog: https://github.com/electron/electron/releases/tag/v37.0.0
- Changelog: https://github.com/electron/electron/releases/tag/v37.1.0

TomaSajt committed:
electron-chromedriver_37: init at 37.1.0

- Changelog: https://github.com/electron/electron/releases/tag/v37.0.0
- Changelog: https://github.com/electron/electron/releases/tag/v37.1.0

mweinelt commented:
> I don't have the time to wait out the entire build, but it reached 1000/43620. I doubt there will be any issues with the build, but who knows...

You do know that we have community builders over in nix-community? https://nix-community.org/community-builders/

TomaSajt commented:
> > I don't have the time to wait out the entire build, but it reached 1000/43620. I doubt there will be any issues with the build, but who knows...
> 
> You do know that we have community builders over in nix-community? https://nix-community.org/community-builders/

Heard about it but never applied for it. Perhaps I should.

TomaSajt commented:
It fails at/after patching shebangs?? Weird...

yuyuyureka committed:
electron-source.electron_37: init at 37.1.0

- Changelog: https://github.com/electron/electron/releases/tag/v37.0.0
- Changelog: https://github.com/electron/electron/releases/tag/v37.1.0

yuyuyureka reviewed (approved):
Built on aarch64-linux (on another machine with less parallelization)

So the build seems to be slightly flaky (possibly related to the high parallelization on the other machine)
